grails-app/controllers/org/pih/warehouse/api/PutawayApiController.groovy

```diff
@@ -107,23 +106,7 @@ class PutawayApiController {
 
         putaway.putawayAssignee = currentUser
 
-        // Bind the putaway items
-        jsonObject.putawayItems.each { putawayItemMap ->
-            PutawayItem putawayItem = new PutawayItem()
-            bindData(putawayItem, putawayItemMap)
-
-            // Bind the split items
-            putawayItemMap.splitItems.each { splitItemMap ->
-                PutawayItem splitItem = new PutawayItem()
-                bindData(splitItem, splitItemMap)
-                putawayItem.splitItems.add(splitItem)
-            }
-
-            putaway.putawayItems.add(putawayItem)
-        }
-
```

Comment:
if you had made sure the binder handles that 2nd level nested (splitItemMap) cases, LGTM